imagebitmap-renderingcontext/WEB_FEAUTRES.yml

```diff
@@ -0,0 +1,3 @@
+features:
+- name: createimagebitmap
```

Comment:
```suggestion
- name: imagebitmaprenderingcontext
```

The `createImageBitmap()` global method is to create an `ImageBitmap` object which isn't the same thing.

Reply:
Ah, thanks!